xsd/netex_framework/netex_genericFramework/netex_zone_support.xsd

```diff
@@ -65,12 +65,34 @@ Rail transport, Roads and Road transport
 			<xsd:documentation>Reference to a ZONE.</xsd:documentation>
 		</xsd:annotation>
 	</xsd:element>
+	<xsd:simpleType name="NameOfRefClassForZoneRefStructure">
+		<xsd:restriction base="NameOfRefClassForGroupOfPointsRefStructure">
+			<xsd:enumeration value="AccessZone"/>
+			<xsd:enumeration value="AdministrativeZone"/>
+			<xsd:enumeration value="FareZone"/>
+			<xsd:enumeration value="GroupOfEntities"/>
+			<xsd:enumeration value="MobilityServiceConstraintZone"/>
+			<xsd:enumeration value="RoutingConstraintZone"/>
+			<xsd:enumeration value="StopArea"/>
+			<xsd:enumeration value="TariffZone"/>
+			<xsd:enumeration value="TariffZoneRef_"/>
```

Comment:
TariffZoneRef_ is a substitution group (this happens multiple times, just pointing it her for the example)